src/grt/test/congestion7.tcl

```diff
@@ -20,5 +20,6 @@ diff_file congestion7.guideok $guide_file
 diff_file congestion7.rptok $rpt_file
 
 # check the congestion progress reports generated
-diff_file congestion7-20.rptok ./results/congestion7-tcl-20.rpt
-diff_file congestion7-40.rptok ./results/congestion7-tcl-40.rpt
+set rpt_folder [file dirname $rpt_file]
+diff_file congestion7-20.rptok [file join $rpt_folder congestion7-tcl-20.rpt]
+diff_file congestion7-40.rptok [file join $rpt_folder congestion7-tcl-40.rpt]
```

Comment:
Why not use make_result_file  ?

Reply:
I think this is clearer/fine: It reuses the make_result_file  to find a file alongside the others.

Fewer assumptions about what make_result_file does.